### PR TITLE
chore: more aggressive caching

### PR DESCRIPTION
**Pull request by tien**
might means the user will need to do a browser refresh sometime to get the latest data

fix Sifchain/issues#1512

### CODE REVIEW

File: app/src/domains/clp/queries/params.ts [hunk 1]
```diff
@@ -20,13 +20,15 @@ export const useRewardsParamsQuery = () => {
       enabled: computed(
         () => sifchainClients.queryClientStatus === "fulfilled",
       ),
+      staleTime: minutesToMilliseconds(5),
```
Comment (intl-man) [inline app/src/domains/clp/queries/params.ts:23]: @tien - is that right? That'll set it to 5 minutes yeah? We want it set to 5-10 seconds (I'd err on the side of caution and go with 10s).

Comment (tien) [reply app/src/domains/clp/queries/params.ts:23]: this is specifically for the rewards params rpc only, which should stay the same every reward period? (7 day ish)

Comment (tien) [reply app/src/domains/clp/queries/params.ts:23]: I can always remove the micro optimization of course since I'm not a fan of doing this either :))

Comment (intl-man) [reply app/src/domains/clp/queries/params.ts:23]: Yes.

Where else are we hitting rpc.sifchain? Ideally those queries should be cached too.

File: app/src/main.ts [hunk 2]
```diff
@@ -25,6 +26,15 @@ console.log(
   import.meta.env.VITE_APP_SHA,
 );
 
-app.use(VueQueryPlugin);
+app.use(VueQueryPlugin, {
+  queryClientConfig: {
+    defaultOptions: {
+      queries: {
+        staleTime: secondsToMilliseconds(10),
```
Comment (tien) [inline app/src/main.ts:33]: @intl-man this is the new default btw

File: app/src/domains/clp/queries/params.ts (hunk 1 above)
Comment (tien) [reply app/src/domains/clp/queries/params.ts:23]: See here https://github.com/Sifchain/sifchain-ui/pull/614/files#r857958561

**Review by alanrsoares — APPROVED** [review]

File: app/src/main.ts (hunk 2 above)
Comment (intl-man) [reply app/src/main.ts:33]: @tien nice!

**Review by intl-man — APPROVED** [review]

**Pull request merged**